systemd/cloud-init.service.tmpl

```diff
@@ -12,6 +12,9 @@ After=systemd-networkd-wait-online.service
 {% if variant in ["ubuntu", "unknown", "debian"] %}
 After=networking.service
 {% endif %}
+{% if variant == "ubuntu" %}
+After=dbus.socket
+{% endif %}
```

Comment:
@TheRealFalcon I actually think this should be placed in cloud-init-local.service as with a `Requires=dbus.socket` to make sure that it is started and running by the time `cloud-init.service` is started. .... In testing it seems to work, but, I'm poking around to prove to  myself why this ordering works as well as discovering why cloud-init.service is getting punted otherwise.

Reply:
As mentioned by Brett, this could add several seconds to boot time:
```
holmanb@elm:~$ systemd-analyze critical-chain dbus.socket
The time when unit became active or started is printed after the "@" character.
The time the unit took to start is printed after the "+" character.

dbus.socket @5.474s
└─sysinit.target @5.377s
  └─cloud-init.service @4.647s +692ms
    └─cloud-init-local.service @1.129s +2.149s
      └─systemd-remount-fs.service @1.095s +31ms
        └─systemd-fsck-root.service @971ms +75ms
          └─systemd-journald.socket @754ms
            └─system.slice @533ms
              └─-.slice @533ms
```